pkg/supportbundle/rbac.go

```diff
@@ -34,34 +34,75 @@ func checkRemoteCollectorRBAC(ctx context.Context, clientConfig *rest.Config, ti
 
 	var forbidden []error
 
-	spec := authorizationv1.SelfSubjectAccessReviewSpec{
-		ResourceAttributes: &authorizationv1.ResourceAttributes{
-			Namespace:   namespace,
-			Verb:        "create,delete",
-			Group:       "",
-			Version:     "",
-			Resource:    "pods,configmap",
-			Subresource: "",
-			Name:        "",
+	resourceAttributesList := []authorizationv1.ResourceAttributes{
+		{
+			Namespace: namespace,
+			Verb:      "get",
+			Resource:  "pods",
+		},
+		{
+			Namespace: namespace,
+			Verb:      "create",
+			Resource:  "pods",
+		},
+		{
+			Namespace: namespace,
+			Verb:      "delete",
+			Resource:  "pods",
+		},
+		{
+			Namespace: namespace,
+			Verb:      "get",
+			Resource:  "pods/log",
+		},
+		{
+			Verb:     "list",
+			Resource: "nodes",
+		},
+		{
+			Namespace: namespace,
+			Verb:      "get",
+			Resource:  "configmaps",
+		},
+		{
+			Namespace: namespace,
+			Verb:      "create",
+			Resource:  "configmaps",
+		},
+		{
+			Namespace: namespace,
+			Verb:      "delete",
+			Resource:  "configmaps",
+		},
+		{
+			Namespace: namespace,
+			Verb:      "get",
+			Resource:  "serviceaccounts",
 		},
-		NonResourceAttributes: nil,
 	}
 
-	sar := &authorizationv1.SelfSubjectAccessReview{
-		Spec: spec,
-	}
-	resp, err := client.AuthorizationV1().SelfSubjectAccessReviews().Create(ctx, sar, metav1.CreateOptions{})
-	if err != nil {
-		return errors.Wrap(err, "failed to run subject review")
-	}
+	for _, resourceAttributes := range resourceAttributesList {
+		spec := authorizationv1.SelfSubjectAccessReviewSpec{
+			ResourceAttributes: &resourceAttributes,
+		}
+
+		sar := &authorizationv1.SelfSubjectAccessReview{
+			Spec: spec,
+		}
+
+		resp, err := client.AuthorizationV1().SelfSubjectAccessReviews().Create(ctx, sar, metav1.CreateOptions{})
```

Comment:
what if the caller does not have `selfsubjectaccessreviews/create` RBAC permission? is it considered an RBACPermissionError?

Reply:
Good question and i'm not sure. I never ran into an issue testing locally with a service account that only fulfilled what is listed above, so i'm not sure if `selfsubjectaccessreviews/create` is implicitly granted by default so that `kubectl auth can-i` can work or what. 

Either way, if you think this is something we need to consider, we'd need to consider it everywhere else we're checking for RBAC as well, so I think its safe to follow that up in a separate issue if you'd like.